lib/package.json

```diff
@@ -2,32 +2,32 @@
   "name": "@storyblok/react",
   "version": "0.0.1",
   "description": "SDK to integrate Storyblok into your project using React.",
-  "main": "./dist/storyblok-react.js",
-  "module": "./dist/storyblok-react.mjs",
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
   "types": "./dist/types/index.d.ts",
   "files": [
     "dist"
   ],
   "exports": {
     ".": {
       "types": "./dist/types/index.d.ts",
-      "import": "./dist/storyblok-react.mjs",
-      "require": "./dist/storyblok-react.js"
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.js"
     },
     "./rsc": {
       "types": "./dist/types/rsc/index.d.ts",
-      "import": "./dist/storyblok-react2.mjs",
-      "require": "./dist/storyblok-react2.js"
+      "import": "./dist/index2.mjs",
```

Comment:
what do you think about naming this one `./dist/index-rsc.mjs`?

Reply:
Right now the name is auto-generated given the file name. If we want this entry to give it a specific name, I'd use `./dist/rsc.mjs` to make it a bit simpler, what do you think?

Reply:
Beautiful! @alexjoverm